de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/session/internal/SarosSession.java

```diff
@@ -563,9 +571,15 @@ public void removeListener(ISessionListener listener) {
         listenerDispatch.remove(listener);
     }
 
+    /*
+     * Set<IProject> projectSet = new HashSet<IProject>(); for (IReferencePoint
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This block of commented-out lines of code should be removed. ([squid:CommentedOutCodeLine](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3ACommentedOutCodeLine))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=210&issues=AWVWjH5glRPS_ea-bTha&open=AWVWjH5glRPS_ea-bTha)

[ISSUE_KEY]: AWVWjH5glRPS_ea-bTha